### PR TITLE
feat(Brave Search): add presence

### PR DESCRIPTION
**Pull request by Kuriel23**
## Description 
Add presence for Brave Search. The presence shows what the user is searching on index, news, images, videos, or goggles and the help pages or settings. Solves #8437 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Discord_2vij8dR4nG

Discord_blKjWDFpA8

Discord_TGQ9oRehCk

Discord_TyvTKGAzAX

</details>

### CODE REVIEW

File: websites/B/Brave Search/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.11",
+	"apiVersion": 1,
+	"author": {
+		"id": "354233941550694400",
+		"name": "kuriel."
+	},
+	"service": "Brave Search",
+	"description": {
+		"en": "The smarter way to search."
```
Comment (darkvillager2) [inline websites/B/Brave Search/metadata.json:10]:
```suggestion
		"en": "Search the Web. Privately. Truly useful results, AI-powered answers, & more. All from an independent index. No profiling, no bias, no Big Tech."
```

File: websites/B/Brave Search/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	},
+	"url": "search.brave.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Oxx2bFw.png",
+	"thumbnail": "https://i.imgur.com/FdxJFrz.png",
+	"color": "#FB542B",
+	"category": "other",
+	"tags": [
+		"search"
+	]
```
Comment (darkvillager2) [inline websites/B/Brave Search/metadata.json:20]: Privacy setting for this is a must imo.

**Review by darkvillager2 — CHANGES_REQUESTED** [review]

**Comment by Kuriel23** [comment]
Done.

**Review by theusaf — APPROVED** [review]

**Review by darkvillager2 — APPROVED** [review]

File: websites/B/Brave Search/presence.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		privacy = await presence.getSetting("privacy"),
+		presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+			details: "In Home",
+			state: privacy
+				? ""
```
Comment (Bas950) [inline websites/B/Brave Search/presence.ts:18]:
```suggestion
				? undefined
```

**Review by Bas950 — APPROVED** [review]

File: websites/B/Brave Search/presence.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		privacy = await presence.getSetting("privacy"),
+		presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+			details: "In Home",
+			state: privacy
+				? // eslint-disable-next-line no-undefined
+				  undefined
```
Comment (darkvillager2) [inline websites/B/Brave Search/presence.ts:19]: no

**Pull request merged**